README.md

```diff
@@ -1,24 +1,27 @@
-# README
+# Turning Green Core
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This repo contains the backend for various Turning Green stuff.
 
-Things you may want to cover:
+## Initial Setup
+For first-time users, do the following
+```
+1. Get Docker
+2. `docker-compose run web rails db:create`
```

Comment:
@fangherk Reading online it seems better to run `docker-compose build` before 2.